embeddings/embedding/static/fasttext.py

```diff
@@ -26,12 +25,6 @@ def __post_init__(self) -> None:
         self.model_name = f"kgr10.plain.{self.method}.dim{self.dimension}.neg10.bin"
 
```

Comment:
in word2vec.py file accessibility is checked: **if not self.file_accessible(self.model_name)**, should it be checked here as well or is it not necessary?

Reply:
it will be added in the next PR